wger.py

```diff
@@ -127,186 +161,188 @@ def create_exercise(self):
         category_id = str([i['id'] for i in categories if i['name'] == category][0])
         equipment_id = str([i['id'] for i in equipments if i['name'] == equipment][0])
         muscle_id =str([i['id'] for i in muscles if i['name'] == muscle][0])
-        data={'name':name, 'description':description, 'category':category_id, 'equipment': equipment_id, 'muscles':muscle_id,
+        data={'name':name,
```

Comment:
Create a dedicated method for creating this exercise(as you did, for example, for add_item).